src/frontend/js/types/payments/lyra.d.ts

```diff
@@ -0,0 +1,3 @@
+interface Window {
```

Comment:
curious: why not just adding this interface directly inside the payment interface? Why do you use a specific `.d.ts`, is there a specific reason?